xml/System.DirectoryServices.Protocols/LdapConnection.xml

```diff
@@ -332,7 +332,10 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>The <see cref="M:System.DirectoryServices.Protocols.LdapConnection.Bind" /> method sends an LDAP bind using the current credentials.</summary>
+        <summary>The <see cref="M:System.DirectoryServices.Protocols.LdapConnection.Bind" /> method sends an LDAP bind using the current credentials.
+        Binding is the step where the LDAP server authenticates the client and, if the client is successfully authenticated, allows the client access to the LDAP server based on that client's privileges.
+        If a connection was created using ldap_connect, and if no binding function is called, on a LDAP v3 server, you run as anonymous.
+        </summary>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <summary>The <see cref="M:System.DirectoryServices.Protocols.LdapConnection.Bind" /> method sends an LDAP bind using the current credentials.</summary>
        <remarks>
          <format type="text/markdown"><![CDATA[
## Remarks

Binding is the step where the LDAP server authenticates the client and, if the client is successfully authenticated, allows the client access to the LDAP server based on that client's privileges.
If a connection was created using `ldap_connect`, and if no binding function is called, on a LDAP v3 server, you run as anonymous.

   ]]></format>
    </remarks>
```

Long details should be put in remarks. Consider also adding link to the relevant section of LDAP documentation.